receiver/sqlserverreceiver/metadata.yaml

```diff
@@ -255,6 +255,139 @@ metrics:
       input_type: string
     attributes: [database.status]
     extended_documentation: This metric is only available when the receiver is configured to directly connect to SQL Server.
+  sqlserver.active.temp.tables:
```

Comment:
@samiura It seems like this would be a good candidate to have some attributes. Counting "active" and "temporary" tables implies that we could also count "inactive" or "non-temporary" tables. 

To better match existing metrics I'd suggest something like `sqlserver.table.count` with an attribute for something like `table_type` (it would probably be ok to have a single value of "temporary" and this could be expanded later) and one like `table_state` (also would probably be fine to have a single value of "active" with any other state added in the future).

Thoughts?

Reply:
I cannot agree more with you. Thank you for the idea.